mpitest/test_mpi_opt.py

```diff
@@ -120,6 +120,7 @@ def test_parab_FD(self):
 
         driver = model.driver = pyOptSparseDriver()
         driver.options['optimizer'] = OPTIMIZER
+        driver.options['print_results'] = False
```

Comment:
Seems like we should standardize: we have `print_results` in pyopt and `disp` in scipyoptimizer. Not a comment on this story though, just something we should fix.